api/types/container/network.go

```diff
@@ -0,0 +1,183 @@
+package container
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// Port is a serialized representation of a network port in the format "80/tcp".
+// It is a string type to allow for easy serialization as a map key (e.g. in [PortSet] and [PortMap]).
+// Valid formats are "80", "80/tcp", "443/udp", "1000-2000/sctp", etc.
+// If no protocol is specified, "tcp" is assumed.
+type Port string
```

Comment:
Probably a better name but effectively `Port` is what a (single) port or port range will serialize as.